NAMESPACE

```diff
@@ -152,6 +157,11 @@ exportMethods(isNotContaminant)
 exportMethods(left_join)
 exportMethods(makePhyloseqFromTreeSE)
 exportMethods(makePhyloseqFromTreeSummarizedExperiment)
+exportMethods(makeTreeSEFromBiom)
+exportMethods(makeTreeSEFromDADA2)
+exportMethods(makeTreeSEFromPhyloseq)
+exportMethods(makeTreeSummarizedExperimentFromBiom)
+exportMethods(makeTreeSummarizedExperimentFromDADA2)
```

Comment:
Why there are these new functions? Only cluster should be added.

Did you check that the functions in deprecate.R matches with the ones that are removed from "main" files?

(The reason seems to be that those functions was not exported. If that is the reason, users have not used these functions --> we can just remove them without deprecation)